daras_ai_v2/copy_to_clipboard_button_widget.py

```diff
@@ -23,20 +22,25 @@ def copy_to_clipboard_button(
     label: str,
     *,
     value: str,
-    style: str = "",
+    style: dict[str, str] | None = None,
     className: str = "",
     type: typing.Literal["primary", "secondary", "tertiary", "link"] = "primary",
-):
-    return gui.html(
-        # language="html"
-        f"""
-<button
-    type="button"
-    class="btn btn-theme btn-{type} {className}"
-    onclick="copyToClipboard(this);"
-    style="{style}"
-    data-clipboard-text="{escape(value)}">
-    {label}
-</button>
-        """,
+    key: str | None = None,
+) -> bool:
+    key = key or "copy-to-clipboard-btn"
```

Comment:
lets not make the other copy buttons slower

Reply:
fixed

Reply:
added a new method `copy_to_clipboard_button_with_return` (had to use chatgpt for name suggestions)